optimizely/optimizely.py

```diff
@@ -1381,3 +1380,5 @@ def close(self) -> None:
         if callable(getattr(self.event_processor, 'stop', None)):
             self.event_processor.stop()  # type: ignore[attr-defined]
         self.odp_manager.close()
+        if callable(getattr(self.config_manager, 'stop', None)):
+            self.config_manager.stop()  # type: ignore[attr-defined]
```

Comment:
What are the benefits of waiting for datafile polling thread stop?

Reply:
Prevents it from throwing errors- mostly during testing